src/Sql/Sql/ChangeLog.md

```diff
@@ -18,11 +18,10 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
-## Version 2.5.0
-* Added readable secondary parameter to `Invoke-AzSqlDatabaseFailover`
-* Added cmdlet `Disable-AzSqlServerActiveDirectoryOnlyAuthentication`
-* Saved sensitivity rank when classifying columns in the database.
+* Add readable secondary parameter to Invoke-AzureSqlDatabaseFailover
+* Add cmdlet Disable-AzSqlServerActiveDirectoryOnlyAuthentication
+* Save sensitivity rank when classifying columns in the database.
+* Add cmdlets Get-AzSqlInstanceOperation and Stop-AzSqlInstanceOperation
```

Comment:
Please manually merge changelog, because above 3 items are in 2.5.0, don't change them. The last item should be put under upcoming release.

Reply:
I did. 
Can we please merge this since after waiting 20 days on the old PR: https://github.com/Azure/azure-powershell/pull/11309 I encountered  major issue while syncing code and I needed to create a new PR.